sagemaker/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM ${SRC_IMAGE}
+USER 0
+RUN apt-get update && apt-get install -y curl
+ENTRYPOINT ["sh", "-c", "curl -L https://bit.ly/nimshim-launch | bash -xe -s -- -c https://bit.ly/nimshim-caddy -e /opt/nim/start-server.sh"]
```

Comment:
This seems bad practice. Is there any way we can avoid curling a file at runtime and instead run this launcher locally?

Reply:
We could statically embed the caddy binary, launch.sh, etc. and allow specification of a custom config via env variable in the sagemaker model config, but the idea was to keep changes to the underlying NIM image as minimal as possible. If SM supported entrypoint overrides and running non-root I would've bypassed a shim image layer entirely (and exclusively provided a proxy layer via this method).

Reply:
Fixed with latest commit @supertetelman